init.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+import argparse
+from argparse import RawTextHelpFormatter
```

Comment:
argparse is fine but since you already are okay with 3rd-party libs, check out

https://click.palletsprojects.com/en/7.x/